xml/ftp_banners.xml

```diff
@@ -904,4 +904,11 @@ against these patterns to fingerprint FTP servers.
     <example>FTP Server Ready</example>
     <example>FTP service ready.</example>
   </fingerprint>
+
+  <fingerprint pattern="^Welcom to ProRat Ftp Server$">
+     <example>Welcom to ProRat Ftp Server</example>
```

Comment:
This failed travis because the `description` element is missing.  For now, just add one.  Longer term the entire concept of validating these files needs to be revisited -- right now it is scattered and inconsistent.